blazor/pdfviewer/getting-started/wsl-application.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: post
+title: Getting Started with PDF Viewer in Blazor WSL mode | Syncfusion
+description: Learn how to getting started with PDF Viewer control in Blazor WSL (Windows Subsystem for Linux) mode. 
+platform: Blazor
+control: PDF Viewer
+documentation: ug
+---
+
+# Getting Started with Blazor PDF Viewer Component in WSL mode
+
+To run the Syncfusion Blazor PDF Viewer in WSL (Windows Subsystem for Linux) mode, you will need to take the following steps:
+
+**Step 1:** Enable the Windows Subsystem for Linux and the Virtual Machine Platform.
+
+![Create-new-blazor-wsl-app](GettingStarted_images/turn-features.png)
+
+To enable the Windows Subsystem for Linux (WSL) and the Virtual Machine Platform on Windows, follow these steps:
+
+Open the Start menu and search for `Control Panel`. Click on `Programs`, then click on `Turn Windows features on or off`. Scroll down and check the box next to `Windows Subsystem for Linux` and `Virtual Machine Platform`. Click `OK` and restart your machine.

```

Comment:
Open the Start menu and search for `Control Panel.` Click on `Programs,` then click on `Turn Windows features on or off.` Scroll down and select the boxes next to `Windows Subsystem for Linux` and `Virtual Machine Platform. Finally, `Click `OK` and restart your machine.